lib/next-amp-demo/pages/components/AmpLightboxGallery.tsx

```diff
@@ -0,0 +1,91 @@
+/**
+ * Copyright 2019 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as React from 'react';
+import {AmpImg, AmpCarousel} from '@ampproject/toolbox-next-amp';
+
+export const config = {amp: true};
+
+export default () => (
+  <main>
+    <h1>amp-image-lightbox</h1>
+    <h2>With AmpImg</h2>
+    <AmpImg
+      lightbox=''
+      src='https://picsum.photos/id/237/300/200'
+      width='300'
+      height='200'
+      layout='responsive'
+    ></AmpImg>
```

Comment:
Do we have a style guide for whether to use `<AmpImg... />` vs `<AmgImg...></AmpImg>`?

Reply:
I'd prefer `<AmpImg... />, I'm only writing the other out of habit...